src/Form/ImportForm.php

```diff
@@ -119,7 +119,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
         try {
           $this->contentLoader->setContentPath($temp_dir);
           $entities = $this->contentLoader->loadContent($temp_name);
-          $this->messenger()->addMessage($this->formatPlural(count($entities), '1 demo entity has been imported.', '@count demo entities have been imported.') . ' Only node and media type entities are listed below - ');
+          $this->messenger()->addMessage($this->t($this->formatPlural(count($entities), '1 demo entity has been imported.', '@count demo entities have been imported.') . ' Only node and media type entities are listed below - '));
```

Comment:
This approach is invalid. `formatPlural()` should not be wrapped inside a `t()`.

The issue is caused by `tide_edit_protection` module hence the fix must be in that module.